data/xml/L12.xml

```diff
@@ -2,13 +2,14 @@
 <collection id="L12">
   <volume id="1">
     <meta>
-      <booktitle>Proceedings of the Eighth International Conference on Language Resources and Evaluation (<fixed-case>LREC</fixed-case>-2012)</booktitle>
+      <booktitle>Proceedings of the Eighth International Conference on Language Resources and Evaluation (<fixed-case>LREC</fixed-case>'12)</booktitle>
       <editor><first>Nicoletta</first><last>Calzolari</last></editor>
       <editor><first>Khalid</first><last>Choukri</last></editor>
       <editor><first>Thierry</first><last>Declerck</last></editor>
       <editor><first>Mehmet Uğur</first><last>Doğan</last></editor>
       <editor><first>Bente</first><last>Maegaard</last></editor>
       <editor><first>Joseph</first><last>Mariani</last></editor>
+      <editor><first>Asuncion</first><last>Moreno</last></editor>
       <editor><first>Jan</first><last>Odijk</last></editor>
```

Comment:
Not present here: http://www.lrec-conf.org/proceedings/lrec2012/index.html

Reply:
Yes they are?

Reply:
Sorry, wrong link (site navigation there is quite bad due to the frames). The link is here: http://www.lrec-conf.org/proceedings/lrec2012/committees.html, where Asuncion appears after Jan.

Reply:
It seems likely they intended alphabetic order, so let's keep that.